packages/combo-box/test/dropdown.test.js

```diff
@@ -40,4 +43,54 @@ describe('overlay', () => {
       expect(loader.nextElementSibling.getAttribute('part')).to.include('content');
     });
   });
+
+  describe('with data provider', () => {
```

Comment:
BTW these tests should be probably moved to `lazy-loading.test.js` as that's where other tests for `dataProvider` are currently located. That file is already quite big, but adding one more suite should be OK.

Reply:
Sure, moved the tests.